src/StdinFileInfo.php

```diff
@@ -46,6 +46,8 @@ public function getATime()
 
     public function getBasename($suffix = null)
     {
+        unset($suffix);
```

Comment:
Trying out the whitelist might work, I did setup it up for a big project. Its like with code coverage, you want to know if things will get worse when a PR is merged, not if the project is in perfect spot.
The comment thing might not be the best either, but it would choose it above no-ops ;)